tests/manage/configuration/test_config.py

```diff
@@ -137,6 +141,36 @@ def test_environment_variable_set_multiple_path(tmp_path):
     assert settings.AIIDA_CONFIG_FOLDER == config_folder
 
 
+@pytest.mark.filterwarnings('ignore:Creating AiiDA configuration folder')
+@pytest.mark.usefixtures('cache_aiida_path_variable')
+def test_cwd(chdir_tmp_path):
+    """Test that if ``AIIDA_PATH`` is not set, the current working directory is checked."""
```

Comment:
I think we also need a test for the opposite case: If AIIDA_PATH _is_ set to a path other then CWD, `.aiida` in CWD should be ignored.
(unless this is effectively tested somewhere already?)

Reply:
I guess every line of the new table in docs in the Examples section should be tested, if not already.

Reply:
I think that is pretty much the case. I added a test that CWD is not matched if the environment variable is set